docs/upgrade_guide.md

```diff
@@ -0,0 +1,30 @@
+# Upgrade Guide
+
+This guide describes how to upgrade Open Service Mesh (OSM).
+
+## Prerequisites
+- Kubernetes cluster with OSM installed
+- The [helm 3 CLI](https://helm.sh/docs/intro/install/)
+
+## Upgrade OSM using Helm
+Use the `helm` CLI to upgrade the OSM control plane from a Kubernetes cluster.
+
+Run `helm upgrade`. 
+```console
+# Add steps for helm upgrade
```

Comment:
@michelleN would love to pair with you on this. I'm not super familiar with the helm commands to get the chart from openservicemesh.github.io/osm via helm CLI